src/mmw/js/src/modeling/models.js

```diff
@@ -170,6 +192,29 @@ var ScenarioModel = Backbone.Model.extend({
 
         this.on('change:project change:name', this.attemptSave, this);
         this.get('modifications').on('add remove', this.attemptSave, this);
+
+        var debouncedPoll = _.debounce(_.bind(this.pollForResults, this), 500);
+        this.get('modifications').on('add change', function() {
```

Comment:
I could be wrong, but you may be able to pass `debouncedPoll` as the second arg directly here.